bootloader/wscript

```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```

Comment:
1. Is this related to msvc ot to `winrc` tool`?
2. Please add a comment, what this does
3. Why do we need to set ` WINRC_SRC_F` to an empty string?
4. These lines should go into the "configure" phase, where the CFLAGS are set - or if it is related to `winrc` into a new part of "tools flags" (after CFLAGS).